tests/rptest/services/flink.py

```diff
@@ -0,0 +1,214 @@
+    # interface, such as 0.0.0.0.
+    "taskmanager.bind-host":
+    "localhost",
+
+    # The address of the host on which the TaskManager runs and can be reached by the JobManager and
+    # other TaskManagers. If not specified, the TaskManager will try different strategies to identify
+    # the address.
+    #
+    # Note this address needs to be reachable by the JobManager and forward traffic to one of
+    # the interfaces the TaskManager is bound to (see 'taskmanager.bind-host').
+    #
+    # Note also that unless all TaskManagers are running on the same machine, this address needs to be
+    # configured separately for each TaskManager.
+    "taskmanager.host":
+    "localhost",
+
+    # The total process memory size for the TaskManager.
+    #
+    # Note this accounts for all memory usage within the TaskManager process, including JVM metaspace and other overhead.
+    "taskmanager.memory.process.size":
+    "1728m",
+
+    # To exclude JVM metaspace and overhead, please, use total Flink memory size instead of 'taskmanager.memory.process.size'.
+    # It is not recommended to set both 'taskmanager.memory.process.size' and Flink memory.
+    #
+    "taskmanager.memory.flink.size":
+    "1280m",
+
+    # The number of task slots that each TaskManager offers. Each slot runs one parallel pipeline.
+    "taskmanager.numberOfTaskSlots":
+    1,
+
+    # The parallelism used for programs that did not specify and other parallelism.
+    "parallelism.default":
+    1,
+
+    #==============================================================================
+    # Rest & web frontend
+    #==============================================================================
+
+    # The port to which the REST client connects to. If rest.bind-port has
+    # not been specified, then the server will bind to this port as well.
+    #
+    "rest.port":
+    8090,
+
+    # The address to which the REST client will connect to
+    "rest.address":
+    "localhost",
+
+    # Port range for the REST and web server to bind to.
+    #rest.bind-port: 8080-8090
+
+    # The address that the REST & web server binds to
+    # By default, this is localhost, which prevents the REST & web server from
+    # being able to communicate outside of the machine/container it is running on.
+    #
+    # To enable this, set the bind address to one that has access to outside-facing
+    # network interface, such as 0.0.0.0.
+    "rest.bind-address":
+    "0.0.0.0",
+
+    # Flag to specify whether job submission is enabled from the web-based
+    # runtime monitor. Uncomment to disable.
+    "web.submit.enable":
+    False,
+
+    # Flag to specify whether job cancellation is enabled from the web-based
+    # runtime monitor. Uncomment to disable.
+    "web.cancel.enable":
+    False,
+}
+
+
+class FlinkService(Service):
+    """
+    Service that runs Job and Task managers on single node
```

Comment:
collect logs for these services? (requires defining a logs dict).

Reply:
Yes, this is in the back log right after implementing simple workload.

Reply:
Done.